ccd-client/src/main/java/uk/gov/hmcts/cmc/ccd/client/model/CaseDataContent.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.cmc.ccd.client.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.Map;
+
+@Data
+@Builder
+public class CaseDataContent {
+    private Event event;
+    private Map<String, JsonNode> data;
+    @JsonProperty("security_classification")
+    private Map<String, JsonNode> securityClassification;
+    @JsonProperty("event_token")
+    private String eventToken;
+    @JsonProperty("ignore_warning")
+    private Boolean ignoreWarning;
```

Comment:
Is that value nullable as use of `Boolean` suggests?

Reply:
I suppose so, as defined by Core Case Data API.

Reply:
can we use a `boolean` instead?

Reply:
done